.github/workflows/test.yml

```
@@ -0,0 +1,94 @@
+---
+    name: "Testing"
+    on:
+      workflow_dispatch:
+        inputs:
+          github-token:
+            type: string
+            description: "GitHub token to use to download artifactory-repo-tool"
+            required: true
+          version:
+            type: string
+            description: "Version of artifactory-repo-tool to install"
+    
+      pull_request:
+      push:
+        branches:
+          - main
+          - 'releases/*'
```

This doesn't match any current branches.  Did you mean `release/*`?
```suggestion
          - 'release/*'
```